plugins/de/opendiabetes/vault/plugin/exporter/plotterexporter/PlotterExporter.java

```diff
@@ -0,0 +1,210 @@
+package de.opendiabetes.vault.plugin.exporter.plotterexporter;
+
+import de.opendiabetes.vault.container.csv.ExportEntry;
+import de.opendiabetes.vault.plugin.exporter.VaultExporter;
+import org.pf4j.Extension;
+import org.pf4j.Plugin;
+import org.pf4j.PluginWrapper;
+
+import java.io.*;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Wrapper class for the PlotterExporter plugin.
+ *
+ * @author ocastx
+ */
+public class PlotterExporter extends Plugin {
+
+    /**
+     * Constructor for the {@link org.pf4j.PluginManager}.
+     *
+     * @param wrapper The {@link org.pf4j.PluginWrapper}.
+     */
+    public PlotterExporter(final PluginWrapper wrapper) {
+        super(wrapper);
+    }
+
+    /**
+     * Actual implementation of the PlotterExporter.
+     */
+    @Extension
+    public static class PlotterExporterImplementation extends VaultExporter {
+
+        /**
+         * Supported plot formats.
+         */
+        private enum PlotFormats {
+
+            /**
+             * Image plot format.
+             */
+            IMAGE,
+
+            /**
+             * LaTeX PDF plot format.
+             */
+            PDF
+        }
+
+        /**
+         * Selected plot format read from the configuration.
+         */
+        private PlotFormats plotFormat;
+
+        /**
+         * Runs the plot script.
+         * @param dataPath path to the data that should be plotted
+         * @param outputPath path where the plot should be written to
+         * @return boolean value indicating whether the command was successful or not
+         */
+        private static boolean plotData(final String dataPath, final String outputPath) {
+            try {
+                Process process = Runtime.getRuntime().exec(new String[]{"python", "ploty.py", "-f", dataPath, "-o", outputPath});
```

Comment:
ploty.py ist plot.py
Und bei mir muss man den pfad von project-root zum Script direkt angeben und das Skript wird nicht mit in die Plugin-ZIP gelegt (genauso wie die config.json)

Reply:
Pfad zum Skript vielleicht über die Config setzen?

Reply:
Am besten mit Java Standardfunktionalität resolven lassen